### PR TITLE
Correct image locations

### PR DESCRIPTION
**Pull request by Wedge009**
Corrects some image paths as flagged by `wmllint`. This complements @pehjota's #8435/#8436 for #8432.

This should probably be back-ported to 1.18 branch.

### CODE REVIEW

**Review by stevecotton — APPROVED** [review]
Not sure who should approve, but it seems more like a case of "somebody did a sanity check" rather than assessing an artistic change.

**Comment by pehjota** [comment]
I saw the bat images change in #6655 and considered including it in my PRs, but I'm kind of glad I didn't since you fixed it along with the Saurian portrait I didn't notice.

Same comment from me: looks sane but I haven't assessed the artistic change (other than knowing that @doofus-01 moved the bat images and Wesnoth is switching from unit PNGs to portrait WebPs).  Yes, it should be backported.

**Comment by Wedge009** [comment]
You might not have noticed the Saurian portraits because there's a built-in fall-back (I think) where if a PNG isn't found, the same file as WebP is looked for. So functionally it should still be the same, but I think it's still good to have the correct reference.

Regarding the bats, it looks to me like a straightforward file move, there's no 'artistic change' that I can tell.

**Comment by stevecotton** [comment]
I meant that this looks like it just needs a sanity check for approval, it doesn't look like there's anything that would need a review from a specific maintainer. Sorry for the confusion.

**Comment by Wedge009** [comment]
No worries, and thanks for the clarification. I was planning to merge fairly soon, in case others wanted to give feedback, but this is pretty straightforward and we seem to be in agreement, so I'll do it now.

**Pull request merged**
